packages/ibm-products-styles/src/components/Datagrid/styles/_datagrid.scss

```diff
@@ -889,9 +970,16 @@
 .#{$block-class} .#{$block-class}__table-row-ai-enabled {
   display: flex;
   align-items: center;
+
+  /* This section to be removed after support for slug dropped  - start */
```

Comment:
same here too, this could be removed in this pr itself, and add corresponding aiLabel classname conditionally on presence of aiLabel || slug prop? can refer alex pr.

Reply:
Some adapters are using slug classes for customising the style for the values passes `slug`. Hence completely removing the classes could break such code. To be on the safer side it would be better to keep the slug classes until be completely remove the support for slug.
I've refactored the style in this section to avoid duplication of styles.